### ISSUE TITLE
Problem after instalation

### ISSUE THREAD

**Issue by plepleple** [tomaciazek/vscode-ansible#6]
### Summary

After installing your VSCode extension and requirements, I am getting error:

![2021-07-18 19_49_03-Window](https://user-images.githubusercontent.com/13676734/126077235-3d6e327b-4b57-4297-983a-1c1a1742cf2a.png)
Syntax error is pointing to _end=_ like this:
![2021-07-18 19_52_55-Window](https://user-images.githubusercontent.com/13676734/126077316-fcb1fcc6-9d1c-439a-ab79-5ebbbb2a1248.png)
Also VSCode points to the syntax error at its beginning:
![2021-07-18 19_49_20-Window](https://user-images.githubusercontent.com/13676734/126077238-05c967a9-5027-410c-a440-12c63c1a2b48.png)

Do you know how to fix it?

### Extension version

1.0.1

### VS Code version

1.58.2 (Windows) + remote to debian host

### Ansible Version

```shell
ansible [core 2.11.2]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/ple/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ple/.local/lib/python3.9/site-packages/ansible
  ansible collection location = /home/ple/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/ple/.local/bin/ansible
  python version = 3.9.2 (default, Apr 12 2021, 15:21:44) [GCC 8.3.0]
  jinja version = 2.11.3
  libyaml = True
```

### OS / Environment

ansible-lint 5.1.2 using ansible 2.11.2

Debian 4.19.194-2 (2021-06-21)

ple@debian:~$ hostnamectl
   Static hostname: debian
         Icon name: computer-vm
           Chassis: vm
    Virtualization: vmware
  Operating System: Debian GNU/Linux 10 (buster)
            Kernel: Linux 4.19.0-17-amd64
      Architecture: x86-64

**Comment by tomaciazek** [comment]
There was indeed a serious bug, substantially degrading functionality, that might have impacted a lot of users.
That syntax error you were seeing was due to Python 2 installed on your OS (which is still quite common).
I've just released a patched version where use of Python 3 is forced. Thanks for reporting!

As for the other error you're seeing - this comes directly from `ansible-lint`, which in turn calls `ansible-playbook --syntax-check`. This means that there is some issue with your playbook that `ansible-lint` and/or `ansible-playbook` can't cope with gracefully.

Try to look at the language server's output (`Output > Ansible Server`) or launching the above-mentioned commands against your playbook, to see what is wrong.
